CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 0.89.0
 
+### Breaking changes
+
+* @PrimaryKey field value can be null for String, Byte, Short, Integer, and Long types. Older Realms should be migrated, using RealmObjectSchema.setNullable() (#2515).
```

Comment:
Older Realms should be migrated using RealmObjectSchema.setNullable() or by adding the @Required annotation.